File: game-backend/src/main/java/com/my/gamebackend/chat/model/dto/ChatRequest.java
package com.my.gamebackend.chat.model.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.LocalDateTime;

/**
 *
 */
@Data
@AllArgsConstructor
@NoArgsConstructor
public class ChatRequest {
    private  String userId; // 요청한 사용자 ID
    private  Long conversationId; // 기존 대화방 ID
    private  String message;    // 사용자가 보낸 메세지

}

File: game_workspace/lib/models/chat/chat_request.dart
class ChatRequest {
  final String userId;
  final String message;
  final int? conversationId;

  ChatRequest(this.userId, this.message, [this.conversationId]);

  Map<String, dynamic> toJson() => {
        'userId': userId,
        'message': message,
        'conversationId': conversationId,
      };
}
